### DISCUSSION TITLE
As the github is provideing Code Scanning to find security vulnerabilities, does the SAST still necessary?

### DISCUSSION THREAD

**Discussion by VampireDaniel** [community/community#53648] [Code Security]
### Select Topic Area

Question

### Body

As GitHub is providing Code Scanning to find security vulnerabilities and coding errors, and maybe Copolit X can help on security as well, so my question would be does the regular SAST stage still necessary?

**Comment by mrepol742** [comment]
SAST is still required to make sure that the code is secure and free of any malicious code, despite the fact that code scanning and Copolit X can help to uncover security flaws and coding problems. 

The faster way to find the issue the better.
